### ISSUE TITLE
UX Improvement: Default scale

### ISSUE THREAD

**Issue by mapninja** [mejackreed/DavidRumseyMapTab#1]
Been using this plugin for a couple of weeks. It's great. I'm finding that most of the maps look better at 1 or 2 zoom levels out from the default, though.

**Comment by mejackreed** [comment]
Yep definitely this is something that needs improvement. One thing I'm trying to balance is zooming in for detail vs showing the whole map with lots of gray space. Maybe a random zoom would be cool?

**Comment by mapninja** [comment]
I think 1 zoom out would be perfect. There's probably no way to make it perfect for every map, since the scales are so varied, but I've found myself clicking the - button at least once in almost every case.
